WebDriverTests/TestExpectations.json

```diff
@@ -3811,20 +3811,34 @@
     "imported/w3c/webdriver/tests/bidi/script/evaluate/user_activation.py": {
         "expected": { "all": { "status": ["FAIL"], "bug": "webkit.org/b/288060"}}
     },
-    "imported/w3c/webdriver/tests/bidi/script/get_realms/context.py": {
-        "expected": { "all": { "status": ["SKIP"], "bug": "webkit.org/b/288061"}}
-    },
     "imported/w3c/webdriver/tests/bidi/script/get_realms/get_realms.py": {
-        "expected": { "all": { "status": ["SKIP"], "bug": "webkit.org/b/288061"}}
+        "subtests": {
```

Comment:
You did a great job documenting the next steps here with links.

Reply:
Thanks! Appreciate the feedback.